Tasks/PublishBuildArtifacts/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,10 +6,8 @@
   "loc.helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=613725)",
   "loc.description": "Publish Build artifacts to the server or a file share using minimatch patterns",
   "loc.instanceNameFormat": "Publish Artifact: $(ArtifactName)",
```

Comment:
If you do a git mv as suggested above then the new task in the same path space will be a git add.  I think that will reflect what really happened.